web-test-runner.config.js

```diff
@@ -1,14 +1,34 @@
 import { esbuildPlugin } from '@web/dev-server-esbuild';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { globbySync } from 'globby';
 
 export default {
+  rootDir: '.',
+  files: 'src/**/*.test.ts',
   concurrentBrowsers: 3,
+  nodeResolve: true,
   plugins: [esbuildPlugin({ ts: true, target: 'auto' })],
+  browsers: [
+    playwrightLauncher({ product: 'chromium' }),
+    playwrightLauncher({ product: 'firefox' }),
+    playwrightLauncher({ product: 'webkit' })
+  ],
   testRunnerHtml: testFramework => `
     <html>
       <head></head>
       <body>
+        <script type="module" src="dist/shoelace.js"></script>
```

Comment:
I mentioned early on that this shouldn't have been removed from the template. Of course, the last file to review explains it. 😅

I still think there's value in keeping some of the common imports in the template. When I create new components, it's nice to have the boilerplate there for tests. It's easy enough to remove if they're not needed.

Reply:
Your comment in the previous discussion explains this, so there's nothing else to do here. Thanks for clarifying that for me!